bundles/EcommerceFrameworkBundle/FilterService/FilterType/ElasticSearch/Input.php

```diff
@@ -23,6 +23,9 @@ class Input extends \Pimcore\Bundle\EcommerceFrameworkBundle\FilterService\Filte
     public function addCondition(AbstractFilterDefinitionType $filterDefinition, ProductListInterface $productList, $currentFilter, $params, $isPrecondition = false)
     {
         $field = $this->getField($filterDefinition);
+        if (!method_exists($filterDefinition, 'getPreSelect')) {
```

Comment:
we should not skip whole method when `getPreSelect` doesn't exist

```suggestion
       $preSelect = null; 
        if (method_exists($filterDefinition, 'getPreSelect')) {
            $preSelect = $filterDefinition->getPreSelect($filterDefinition);
        } 
```

Reply:
still open